Dfe.Academies.Academisation.Domain/ProjectAggregate/Project.cs

```diff
@@ -295,7 +296,7 @@ public CommandResult Update(ProjectDetails detailsToUpdate)
 
 	private static string? ToYesNoString(bool? value)
 	{
-		if (!value.HasValue) return null;
+		if (value.HasValue is false) return null;
```

Comment:
Don't mind the change but does it improve anything?

Reply:
Readablity :)